docs/configuration/intra_node_data_path.md

```diff
@@ -183,3 +190,75 @@ On the other hand, CARET requires users to provide the following description if
 
 User have to fill callback name in `variable_passings`, `publishes`'s `callback_name`. `context_type` must be set as `callback_chain`.
 After editing, use path.verify() described in the beginning of this section to verify that it has been set correctly.
+
+### Manipulating the object
+
+CARET serves Python-based APIs to define intra-node data path.
```

Comment:
```suggestion
CARET serves Python API to define intra-node data path.
```

Reply:
Fixed it in [fb2c5ab](https://github.com/tier4/CARET_doc/pull/192/commits/fb2c5ab09719329bb9546fcdba69c3295f8991f2).